source/manual/email-notifications-how-they-work.html.md

```diff
@@ -63,8 +63,9 @@ endpoint is exposed publicly through
 https://email-alert-api-public.publishing.service.gov.uk.
 
 You can login to the Notify account by going to
-[https://www.notifications.service.gov.uk](). The login credentials are
-in the [2nd line password store][password-store] under `govuk-notify/govuk-email-courtesy-copies`.
+[https://www.notifications.service.gov.uk](https://www.notifications.service.gov.uk). The
```

Comment:
Can we change this to `[login to the Notify account](https://www.notifications.service.gov.uk)` maybe?